src/components/LoansByCategory/HelpmeChoose/HelpmeChooseRecommendations.vue

```diff
@@ -27,7 +27,11 @@
 			/>
 
 			<!-- eslint-disable max-len -->
-			<button @click="goBack" class="tw-hidden lg:tw-flex tw-items-center tw-justify-center tw-gap-1 tw-text-action tw-text-small tw-mt-1">
+			<button
+				@click="goBack"
+				class="tw-hidden lg:tw-flex tw-items-center tw-justify-center tw-gap-1 tw-text-action tw-text-small tw-mt-1"
+				v-kv-track-event="['event-tracking', 'click', 'help-choosing-click-go-back']"
```

Comment:
This event should be: 'event-tracking', 'click', 'help-choosing-go-back' we already know it is a click because of the event option, so 'help-choosing-click-go-back' is redundant -- same for the next instance of this event

Reply:
Thank you @eddieferrer, letting know to the team and fixing now.